mycroft/client/speech/__main__.py

```diff
@@ -35,24 +35,32 @@
 def handle_record_begin():
     """Forward internal bus message to external bus."""
     LOG.info("Begin Recording...")
-    bus.emit(Message('recognizer_loop:record_begin'))
+    context = {'client_name': 'mycroft_listener',
+               'source': 'audio'}
+    bus.emit(Message('recognizer_loop:record_begin', context=context))
 
 
 def handle_record_end():
     """Forward internal bus message to external bus."""
     LOG.info("End Recording...")
-    bus.emit(Message('recognizer_loop:record_end'))
+    context = {'client_name': 'mycroft_listener',
+               'source': 'audio'}
+    bus.emit(Message('recognizer_loop:record_end', context=context))
 
 
 def handle_no_internet():
     LOG.debug("Notifying enclosure of no internet connection")
-    bus.emit(Message('enclosure.notify.no_internet'))
+    context = {'client_name': 'mycroft_listener',
+               'source': 'audio'}
```

Comment:
very minor nitpick, and not used in practice, but should i set destination here as "enclosure"?